pkg/operator/controller/ingress/load_balancer_service.go

```diff
@@ -794,7 +794,7 @@ func loadBalancerServiceIsProgressing(ic *operatorv1.IngressController, service
 	if wantScope != haveScope {
 		err := fmt.Errorf("The IngressController scope was changed from %q to %q.", haveScope, wantScope)
 		switch platform.Type {
-		case configv1.AWSPlatformType, configv1.IBMCloudPlatformType:
+		case configv1.AWSPlatformType, configv1.IBMCloudPlatformType, configv1.PowerVSPlatformType, configv1.AlibabaCloudPlatformType, configv1.OpenStackPlatformType:
```

Comment:
Can this logic use `platformsWithMutableScope` so that the logic can never become inconsistent?

Reply:
Ah yes good idea. Let me look into that.

Reply:
Done.